drivers/all_neighbors_system_dashboard/app/views/all_neighbors_system_dashboard/warehouse_reports/reports/_unhoused_population.haml

```diff
@@ -0,0 +1,108 @@
+- start_date = (DateTime.current - 3.years).beginning_of_month
+- end_date = DateTime.current.beginning_of_month
```

Comment:
Same with the dates here, just flagging so when we get real data we're aware they may need to come from somewhere else.  (I'm thinking report start and report end.)